### PR TITLE
qa_crowbarsetup: Allow vxlan for linuxbridge

### PR DESCRIPTION
**Pull request by vuntz**
This depens on https://github.com/crowbar/crowbar-openstack/pull/424

### CODE REVIEW

**Comment by bmwiedemann** [comment]
+1

**Comment by toabctl** [comment]
+1

**Comment by bmwiedemann** [comment]
mkcloud test with ovs+gre passed, so I kicked off another one with linuxbridge+vxlan

**Comment by vuntz** [comment]
> so I kicked off another one with linuxbridge+vxlan

Well, it won't pass since https://github.com/crowbar/crowbar-openstack/pull/424 isn't merged...

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -2716,7 +2716,11 @@ function custom_configuration()
             fi
         ;;
         neutron)
-            [[ "$networkingplugin" = linuxbridge ]] && networkingmode=vlan
+            if iscloudver 7plus; then
+              [[ "$networkingplugin" = linuxbridge && "$networkingmode" = gre ]] && networkingmode=vlan
+            else
+              [[ "$networkingplugin" = linuxbridge ]] && networkingmode=vlan
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: need 2 extra spaces of indention here and 2 lines above for bashate

Comment (vuntz) [inline scripts/qa_crowbarsetup.sh]: done

**Comment by bmwiedemann** [comment]
for some reason https://ci.suse.de/job/openstack-mkcloud/26513/ passed without it

**Comment by nicolasbock** [comment]
+1

**Pull request merged**